ibm_catalog.json

```diff
@@ -65,7 +65,7 @@
               }
             ]
           },
-          "release_notes_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/releases",
+          "release_notes_url": "https://test.cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-release-notes",
```

Comment:
This link will need to be updated to https://cloud.ibm.com/docs-draft/watsonx-ai-saas-automation?topic=watsonx-ai-saas-automation-release-notes when we push to production.

Reply:
Why `docs-draft`? Shouldn't be `docs`?